lib/feature/detail_card/detail_card_bloc.dart

```diff
@@ -472,24 +480,61 @@ class DetailCardBloc extends BlocBase {
     appBarEnumSubject.value = DetailCardAppBarEnum.comment;
   }
 
-  void sendComment() {
+Future<void> sendComment() async {
     if (!isChatCommentSubject.value) {
       return;
     }
-    final comment = Fragment$CommentFragment(
-      id: '1',
-      createdAt: '',
-      user: Fragment$UserFragment(
-        uid: '1',
-        fullName: 'Đinh Viết Khang',
-        createdAt: DateTime.now().toIso8601String(),
+    isLoadingAddCommentSubject.value = true;
+    await graphqlService.client.mutate$CreateComment(
+      Options$Mutation$CreateComment(
+        variables: Variables$Mutation$CreateComment(
+          idCard: cardSubject.value!.id,
+          content: commentController.text,
+        ),
+      ),
+    );
+    final resultGetCard = await graphqlService.client.mutate$GetCard(
+      Options$Mutation$GetCard(
+        variables: Variables$Mutation$GetCard(
+          idCard: idCard,
+        ),
+      ),
+    );
```

Comment:
```suggestion
    final resultGetCard = await graphqlService.client.mutate$GetCard(
      Options$Mutation$GetCard(
        variables: Variables$Mutation$GetCard(
          idCard: idCard,
        ),
      ),
    );
```
xử lý phía client thôi để tối ưu hiệu năng, đừng call load lại comment nữa, nếu nó xảy ra lỗi mới gọi lại api trên